index.js

```diff
@@ -96,24 +189,119 @@ var app = new Vue({
              * changed array is used to track which categories were already modified to avoid double assigments 
              * and leaving other categories unchanged
              */
-            AttrAllowed = [7,6,5]
-            AttrSums = [
-                app.Attributes[0].sum,
-                app.Attributes[1].sum,
-                app.Attributes[2].sum
+            var AttrAllowed = [7,6,5]
+            var AttrSums = [
+                app.form.attributes[0].sum,
+                app.form.attributes[1].sum,
+                app.form.attributes[2].sum
             ]            
             AttrSums.sort(function(a,b) {return a-b}).reverse()
-            changed = []
+            var changed = []
             while (AttrSums.length > 0) {
-                for (i=0; i < app.Attributes.length; i++) {
-                    if (app.Attributes[i].sum == AttrSums[0] && !(changed.includes(i))) {
-                        app.Attributes[i].allowed = AttrAllowed[0]
+                for (var i=0; i < app.form.attributes.length; i++) {
+                    if (app.form.attributes[i].sum == AttrSums[0] && !(changed.includes(i))) {
+                        app.form.attributes[i].allowed = AttrAllowed[0]
                         AttrSums.shift()
                         AttrAllowed.shift()
                         changed.push(i)
                     }
                 }
             }  
-        }
+        },
```

Comment:
dalej to funguje tak, ze 
- sheet najprv prekreslim do blank sheetu, kde prepisem iba existujuce hodnoty. Spravim si ale kopiu blanku cez cloneArray.
- ulozi sa to do blank premenej
- potom to parsujem do struktury, vegenerujem ju podla data.form_structure

a potom by sa malo spravit prepocitanie sum+priorit, to tu nemam, lebo pouzivam iny kod. Tak to niekto dorobte. Je tam zakomentovana funkcia.

Reply:
function cloneArray is not used anywhere in the code, there is just definition